src/lib.rs

```diff
@@ -11,7 +11,15 @@ use router::{Match, Router};
 
 #[derive(Default)]
 pub struct RouteBuilder {
-    routers: HashMap<Method, Router<Box<dyn Handler>>>,
+    routers: HashMap<Method, Router<WrappedHandler>>,
+}
+
+#[derive(Clone)]
+struct RoutePattern(&'static str);
```

Comment:
I think this needs to be `pub` so that we can extract this from the typemap in our crates.io middleware.